scripts/quests/sandoria/DRG_AF2_Chasing_Quotas.lua

```diff
@@ -0,0 +1,272 @@
+-- Esca !gotoid 17187493
+-- Batallia Downs ??? (qm2) !gotoid 17207822
+
+-- For era purposes, will need to change the timer to getMidnight()
+-----------------------------------
+require('scripts/globals/items')
+require("scripts/globals/keyitems")
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require("scripts/globals/status")
+require('scripts/globals/interaction/quest')
+-----------------------------------
+local batalliaID = require("scripts/zones/Batallia_Downs/IDs")
+local westRonfaureID = require("scripts/zones/West_Ronfaure/IDs")
+local portSandyID = require('scripts/zones/Port_San_dOria/IDs')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.SANDORIA, xi.quest.id.sandoria.CHASING_QUOTAS)
+
+quest.reward =
+{
+    item     = xi.items. DRACHEN_BRAIS,
+    fame     = 40,
+    fameArea = xi.quest.fame_area.SANDORIA,
+}
+
+quest.sections =
+{
+    -- Section: Quest available.
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:getMainJob() == xi.job.DRG and
+                player:getMainLvl() >= xi.settings.main.AF2_QUEST_LEVEL and
+                player:getQuestStatus(xi.quest.log_id.SANDORIA, xi.quest.id.sandoria.A_CRAFTSMAN_S_WORK) == QUEST_COMPLETED
+        end,
+
+        [xi.zone.PORT_SAN_DORIA] =
+        {
+            ['Arminibit'] =
+            {
+                onTrigger = function(player, npc)
+                    if player:getVar("Quest[0][95]csSeen") == 0 then
+                        return quest:progressEvent(18)
+                    elseif player:getVar("Quest[0][95]csSeen") == 1 then --Declined First Offering.
+                        return quest:progressEvent(14)
+                    end
+                end,
+            },
+
+            ['Ceraulian'] =
+            {
+                onTrigger = function(player, npc)
+                    if player:getVar("Quest[0][95]csSeen") == 0 then
+                        return quest:progressEvent(18)
+                    elseif player:getVar("Quest[0][95]csSeen") == 1 then --Declined First Offering.
+                        return quest:progressEvent(14)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [14] = function(player, csid, option, npc)
+                    if option == 1 then
+                        quest:begin(player)
+                        quest:setVar(player, 'csSeen', 0)
+                    else
+                        quest:setVar(player, 'csSeen', 1) -- Declined First Offering.
+                    end
+                end,
+
+                [18] = function(player, csid, option, npc)
+                    if option == 1 then -- Accept quest.
+                        quest:begin(player)
+                        quest:setVar(player, 'csSeen', 0)
+                    else
+                        quest:setVar(player, 'csSeen', 1)
```

Comment:
Same things I said about the first quest apply in this one too.

- `quest:getVar(player, 'csSeen')` instead of `player:getVar("Quest[0][95]csSeen")`
- In this case, I see a timer variable, so we cant get away with just 1 progression variable. Probably